### PR TITLE
[TS] LPS-164298 changing catch to NoSuchFileEntryException as NoSuchImageException is not being thrown, avoid sending default error when NSFE is thrown as 404 is being sent already

### PR DESCRIPTION
**Pull request by jgarcialr**

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/util/PortalImpl.java [hunk 1]
```diff
@@ -6663,7 +6663,11 @@ else if (PropsValues.LAYOUT_SHOW_HTTP_STATUS) {
 
 			httpServletResponse.setStatus(status);
 
-			SessionErrors.add(httpSession, exception.getClass(), exception);
+			if ((exception != null) &&
+				!(exception instanceof NoSuchFileEntryException)) {
```
[the remark this answers is not in the document]
Comment (jgarcialr) [reply portal-impl/src/com/liferay/portal/util/PortalImpl.java:6667]: Hi @dantewang ,
There's a forwarding if the request is made directly from the browser, per example, when you access the URL of the document. 
If the request is made within the page, via fragment or content display, there's no forward and the error stays in session errors until a new request is made.

[the remark this answers is not in the document]
Comment (jgarcialr) [reply portal-impl/src/com/liferay/portal/util/PortalImpl.java:6667]: Hi @dantewang ,

If I debug, when pausing on the requests I can see the redirection to `/c/portal/status`, it's shown in the html title, In this case no error is shown. But in a normal flow, I'm not able to get the redirection to the status page and got the error message.

**Comment by catarineaguiar** [comment]
@jgarcialr no regressions found.

**Comment by tinatian** [comment]
@jgarcialr 
`(exception != null) && !(exception instanceof NoSuchFileEntryException)` This is duplicated, instanceof will also does null checking.
@vicnate5 Do we have tests covered this part?

**Comment by Kvale1733** [comment]
Resent with test here: https://github.com/liferay-core-infra/liferay-portal/pull/1972

**Comment by vicnate5** [comment]
ci:close

**Pull request closed without merging**